server/src/main/java/edu/jhu/server/Board.java

```diff
@@ -63,7 +63,7 @@ public Board() {
     }
 
     String[] suspects = {"colonel_mustard", "miss_scarlet", "professor_plum", "mr_green",
-        "mrs_white", "mrs_peacock"};
+        "mrs_white", "Mrs. Peacock"};
```

Comment:
yeah if i had been smart, i would've made the names static String variables and then we'd only have to change them in one place... woops